Assets/Game/Scripts/Game/ExplosionButton.cs

```diff
@@ -1,14 +1,35 @@
-﻿using UnityEngine;
+using ChrisNolet.QuickOutline;
+using UnityEngine;
+using UnityEngine.Events;
 
 namespace GameJammers.GGJ2025.Bootstraps {
     public class ExplosionButton : MonoBehaviour {
         GameController _game;
+        private Outline buttonoutline;
 
         void Start () {
             _game = GameController.Instance;
+            TryGetComponent(out buttonoutline);
+            if(buttonoutline != null) {
+                buttonoutline.enabled = false;
+            }
+        }
+
+        void OnMouseEnter () {
+            if (buttonoutline == null) { return; }
+            buttonoutline.enabled = true;
```

Comment:
Should be safe to simplify this.

```suggestion
            buttonoutline?.enabled = true;
```